### ISSUE TITLE
Only set the graph axis name when it's manually set

### ISSUE THREAD

**Issue by par456** [APSIMInitiative/ApsimX#8784]
### Describe the new feature

When a graph is created, it often sets the axis name into the axis that is then stored, even if the axis was not manually named. This then causes the name to be copied when the graph is copied and doesn't update when the X or Y axis value is changed. Leading to graphs with incorrectly named axis when you are making a lot at once and working from copies.

Should fix this so that the axis name is only saved if it was actually manually renamed, otherwise it should continue to change based on what the variable it has been set to.

Might need to also clear old graphs that have an axis with the generated name stored so that older files share the behaviour.

**Comment by par456** [comment]
Wrong PR linked, ignore that.

**Comment by ric394** [comment]
Was unable to replicate the behaviour mentioned. Has this been fixed with another PR @par456?
Feel free to reopen if I'm incorrect, but I'll close this as it doesn't seem to be an issue on tip.

**Comment by par456** [comment]
The issue may have been fixed, but there are a lot of old graphs hanging around with locked axis names due to the text being saved.

So the 2nd part of this issue is to write some code (maybe in the converter) to clear a graph's axis name if the name it has is the same one that the graph system would automatically generate. That way we can 'unset' some of these axis that have been incorrectly set to a name.

**Comment by AdamLiedloff** [comment]
I have just come across this issue and wondered when this will be addressed as I copy many graphs as it has the tree structure with series ready to be set. It didn't worry me but other users are confused when the axis label is wrong (i.e. $ not kg). So whenever a series axis is changed where the label was previously set, this too should remove the manual label.

**Comment by lie112** [comment]
I have just come across this issue and wondered when this will be addressed as I copy many graphs as it has the tree structure with series ready to be set. It didn't worry me but other users are confused when the axis label is wrong (i.e. $ not kg). So whenever a series axis is changed where the label was previously set, this too should remove the manual label.

**Comment by par456** [comment]
Agreed, that seems like a sensible solution. I'll see we can implement it that way.
